packages/plugin-core/package.json

```diff
@@ -1115,6 +1115,7 @@
     "replace-in-file": "^6.2.0",
     "semver": "^7.3.2",
     "string-similarity": "^4.0.4",
+    "textextensions": "^5.14.0",
```

Comment:
can we do a callout in the PR when we add dependencies?

just created an async around it in -> https://github.com/dendronhq/dendron-docs/commit/4998410128feefdf59d108577b100fe88c11c55f